.github/workflows/buildReleaseAndPublish.yml

```diff
@@ -22,7 +22,7 @@ jobs:
     permissions:
       contents: write
     # Don't run this in everyone's forks.
-    if: github.repository == 'llvm/torch-mlir-release'
+    if: ${{ github.repository_owner == 'llvm' || github.event_name != 'schedule' }}
```

Comment:
IIUC you want this to run on schedule in main repo, and explicit dispatch on forks. Could you perhaps add a breadcrumb/comment to clarify?

Reply:
Yes, as result scheduled jobs are only run in `llvm/torch-mlir` but a user can run the workflow via explicit dispatch. I updated the comment to reflect this.